honeybee_radiance_folder/gridutil.py

```diff
@@ -182,7 +192,10 @@ def get_target_file(index):
             total_count, grid_count, sensor_per_grid
             )
     )
-    return grid_count, sensor_per_grid
+    if return_out_grid_info:
+        return grid_count, sensor_per_grid, out_grid_info
+    else:
+        return grid_count, sensor_per_grid
```

Comment:
Now I see the challenge here! Even though it is a breaking change it is better to always return 3 outputs and fix it in the code-base wherever it is used. I don't think we use it in any client-facing component. We can fix it inside the `honeybee-radiance` library without breaking anything for the users.

Reply:
Okay. I have removed the 'return_out_grid_info' option so now it will always output this. As far as I can see we only use this function here: https://github.com/ladybug-tools/honeybee-radiance/blob/master/honeybee_radiance/cli/grid.py#L285